Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/wrapper/impl/ConceptEntryWrapperCreator.java

```diff
@@ -36,8 +41,12 @@
     private IUserManager usersManager;
 
     @Autowired
-    private IURIHelper helper;
-
+    private IURIHelper helper; 
+    
+    @Autowired
+    @Qualifier("conceptReviewDatabaseManager")
+    private DatabaseManager dbManager;
```

Comment:
the databaseManager should only be used in the database connection class, nowhere else.